plutus-tx-plugin/test/Budget/9.6/sumAtIndices.pir.golden

```diff
@@ -0,0 +1,16 @@
+let
+  !tail-1935 :
+     all a-1936. (\a-1937 -> list data) a-1936 -> (\a-1938 -> list data) a-1936
+    = /\a-1932 ->
+        \(eta-1933 : (\a-1934 -> list data) a-1932) -> tailList {data} eta-1933
```

Comment:
I suggest you rebase this PR onto you recent inliner fix for this one to get inlined.
This file will become
```
\(d-1579 : data) ->
  let
    !nt-1580 : list data = tailList {data} (unListData d-1579)
    !s-1581 : integer = unIData (headList {data} nt-1580)
    !nt-1582 : list data
      = tailList {data} (tailList {data} (tailList {data} nt-1580))
    !s-1583 : integer = unIData (headList {data} nt-1582)
    !s-1584 : integer = unIData (headList {data} (tailList {data} nt-1582))
  in
  addInteger (addInteger s-1581 s-1583) s-1584
  ```
  after rebase.

Reply:
done